README.md

```diff
@@ -28,6 +28,7 @@ If no log is used, a temporary log name is created by the resource.
 * **UpdateNow**: Indicates if the resource should trigger an update during consistency (including initial.)
 * **Category**: Indicates the categories (one or more) of updates the resource should update for.  'Security', 'Important', 'Optional'.  Default: 'Security' (please note that security is not mutually exclusive with Important and Optional, so selecting Important may install some security updates, etc.)
 * **Notifications**: Sets the windows update agent notification setting.  Supported options are 'disabled' and 'ScheduledInstallation'.  [Documentation from Windows Update](https://msdn.microsoft.com/en-us/library/windows/desktop/aa385806%28v=vs.85%29.aspx?f=255&MSPPError=-2147217396)
+* **RecommendedUpdates**: Whether to include Recommended updates (default false).
```

Comment:
This description is not accurate.  Please update the description.  Please include the impact on `Optional` and `Important` updates.